packages/quantic/force-app/main/default/lwc/quanticTabBar/e2e/quanticTabBar.e2e.ts

```diff
@@ -0,0 +1,199 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+const extraSmallViewportWidth = 200;
+const smallViewportWidth = 260;
+const mediumViewportWidth = 300;
+const standardViewportHeight = 1080;
+
+const viewportWidths = [
+  {
+    label: 'extra small',
+    width: extraSmallViewportWidth,
+    primaryExpectation: 'the selected tab ',
+    secondaryExpectation: 'and the more button should be shrunk',
```

Comment:
Superfluous to create two variables there

```suggestion
    expectationDescription: 'should display only the selected tab, the other tabs should be displayed in the dropdown list and the more button should be shrunk'
```